localstack/services/events/provider_v2.py

```diff
@@ -58,18 +148,185 @@ def put_rule(
+            tags,
+            event_bus_name,
+            targets,
         )
+        self._rule_workers[rule_worker.arn] = rule_worker
+        event_bus.rules[name] = rule_worker.rule
+        response = PutRuleResponse(RuleArn=rule_worker.arn)
         return response
+
+    @handler("DeleteRule")
+    def delete_rule(
+        self,
+        context: RequestContext,
+        name: RuleName,
+        event_bus_name: EventBusNameOrArn = None,
+        force: Boolean = None,
+        **kwargs,
+    ) -> None:
+        store = self.get_store(context)
+        event_bus_name = self._extract_event_bus_name(event_bus_name)
+        event_bus = self.get_event_bus(event_bus_name, store)
+        if rule := event_bus.rules.get(name):
+            rule_worker = self._rule_workers[rule.arn]
+            rule_worker.delete()
+            self._rule_workers.pop(name)
+            # self._event_bus_workers[event_bus_name].rules.pop(name)
+            del event_bus.rules[name]
+        else:
+            return
+
+    @handler("ListRules")
+    def list_rules(
+        self,
+        context: RequestContext,
+        name_prefix: RuleName = None,
+        event_bus_name: EventBusNameOrArn = None,
+        next_token: NextToken = None,
+        limit: LimitMax100 = None,
+        **kwargs,
+    ) -> ListRulesResponse:
+        return {"Rules": []}  # TODO implement
+
+    @handler("ListRuleNamesByTarget")
+    def list_rule_names_by_target(
+        self,
+        context: RequestContext,
+        target_arn: TargetArn,
+        event_bus_name: EventBusNameOrArn = None,
+        next_token: NextToken = None,
+        limit: LimitMax100 = None,
+        **kwargs,
+    ) -> ListRuleNamesByTargetResponse:
+        raise NotImplementedError  # TODO implement
+
+    @handler("DescribeRule")
+    def describe_rule(
+        self,
+        context: RequestContext,
+        name: RuleName,
+        event_bus_name: EventBusNameOrArn = None,
+        **kwargs,
+    ) -> DescribeRuleResponse:
+        raise NotImplementedError  # TODO implement
+
+    @handler("DisableRule")
+    def disable_rule(
+        self,
+        context: RequestContext,
+        name: RuleName,
+        event_bus_name: EventBusNameOrArn = None,
+        **kwargs,
+    ) -> None:
+        raise NotImplementedError  # TODO implement
+
+    @handler("EnableRule")
+    def enable_rule(
+        self,
+        context: RequestContext,
+        name: RuleName,
+        event_bus_name: EventBusNameOrArn = None,
+        **kwargs,
+    ) -> None:
+        raise NotImplementedError  # TODO implement
+
+    def get_store(self, context: RequestContext) -> EventsStore:
+        region = context.region
+        account_id = context.account_id
+        store = events_store[account_id][region]
+        # create default event bus on first call
+        name = "default"
+        if name not in store.event_buses.keys():
+            event_bus_worker = EventBusWorker(name, region, account_id)
```

Comment:
does this differ from `create_event_bus_worker`?

Reply:
fixed in: https://github.com/localstack/localstack/pull/10613/commits/5f7c3d0523d0229906cc6b58036378c9b91fdc54